src/safeds/ml/classical/classification/_random_forest.py

```diff
@@ -13,9 +13,23 @@
 
 
 class RandomForest(Classifier):
-    """Random forest classification."""
-
-    def __init__(self) -> None:
+    """Random forest classification.
+
+    Parameters
+    ----------
+    number_of_trees : int
+        The number of trees to be used in the random forest. Has to be greater than 0.
+
+    Raises
+    ------
+    ValueError
+        If the number of trees is less than 1.
```

Comment:
```suggestion
        If the number of trees is less than or equal to 0.
```
I think it is better to understand if you use the same value in the docs (required greater than 0 -Error if less than or equal to 0)